.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [master]
```

Comment:
To confirm, we're releasing on any tag and any push to master?

Reply:
Yes.